src/main/java/Deadline.java

```diff
@@ -0,0 +1,21 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+    private LocalDateTime time;
+
+    public Deadline(String command, LocalDateTime time) {
+        super(command);
+        this.time = time;
+    }
+
+    public LocalDateTime getTime() {
+        return this.time;
+    }
```

Comment:
According to the module's coding standards, should a javadoc comment exist above the method? The same comment for all other public methods.